content/_index.html

```diff
@@ -138,13 +138,16 @@ <h3 class="mt-6 text-white">Install with Conda</h3>
   bash Miniconda3-latest-Linux-x86_64.sh</code></pre>
               <br>
               <p><strong>2.</strong> Then quick install RAPIDS with:</p>
-              <pre class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-23.10 -c rapidsai -c conda-forge -c nvidia rapids=23.10 python=3.10 cudatoolkit=11.8</code></pre>
+              <pre class="highlight use-white-copy"><code>conda create --solver=libmamba -n rapids-23.12 -c rapidsai -c conda-forge -c nvidia rapids=23.12 python=3.10 cudatoolkit=11.8</code></pre>
 
               <h3 class="mt-6 text-white">Install with pip</h3>
               <p><strong>1.</strong> Install via pip channels:</p>
```

Comment:
This is a single-step installation. We can remove the **1.**
```suggestion
              <p>Install via pip channels:</p>
```